de.dashup.application/src/main/webapp/components/dashup-display.js

```diff
@@ -0,0 +1,29 @@
+import {LitElement, html, css} from "https://unpkg.com/lit-element@2.1.0/lit-element.js?module";
+import {DashupComponent} from "./dashup-component.js";
+export class DashupDisplay extends DashupComponent{
```

Comment:
Why `DashupDisplay`? A display is something like a monitor or so. I'd change the name to `DashupLabel`

Reply:
Well, but a label is something that adds descriptive data to other data. The name display is chosen, because the component "displays" data and adds a label to it.